<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicLrcArtPhotoUpload</class>
 <widget class="QWidget" name="MusicLrcArtPhotoUpload">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MusicLrcArtPhotoUpload{border:1px solid gray}</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>415</width>
     <height>320</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>上传歌手写真</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>2</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="artNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>42</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>歌手：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="artSearchEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>42</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="stateLabel">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>285</y>
     <width>411</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>选择的图片尺寸过小，请重新选择</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="uploadButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="MusicLrcArtPhotoLabel" name="imageLabel" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>341</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>点击选择图片</string>
   </property>
  </widget>
  <widget class="QLabel" name="introduceLabel">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>170</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>请上传尺寸1001*669以上的图片</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>25</y>
     <width>415</width>
     <height>295</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,150)</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>artNameLabel</zorder>
  <zorder>artSearchEdit</zorder>
  <zorder>stateLabel</zorder>
  <zorder>uploadButton</zorder>
  <zorder>closeButton</zorder>
  <zorder>imageLabel</zorder>
  <zorder>selectButton</zorder>
  <zorder>introduceLabel</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicLrcArtPhotoLabel</class>
   <extends>QWidget</extends>
   <header location="global">lrcmanager/musiclrcartphotolabel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
